src/libstd/fs.rs

```diff
@@ -844,6 +844,9 @@ pub fn symlink_metadata<P: AsRef<Path>>(path: P) -> io::Result<Metadata> {
 
 /// Rename a file or directory to a new name.
 ///
+/// Moving a file or directory is also a rename, in a sense, and so this
+/// function is not limited a single directory.
```

Comment:
Yeah, I think there was a filing of an issue, and a discussion at the same time. Shame the IRC stuff is lost, argh. We should mention the FS issue regardless.